src/app/series/series.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Subscription } from 'rxjs';
+
+import {
+  ButtonComponent,
+  CmsService,
+  FancyFieldComponent,
+  HeroComponent,
+  ModalService,
+  SeriesModel
+} from '../shared';
+
+@Component({
+  directives: [HeroComponent, ButtonComponent, FancyFieldComponent],
+  selector: 'publish-series',
+  styleUrls: ['series.component.css'],
+  templateUrl: 'series.component.html'
+})
+
+export class SeriesComponent implements OnInit, OnDestroy {
+
+  private id: number;
+  private series: SeriesModel;
+  private routeSub: Subscription;
+
+  constructor(
+    private cms: CmsService,
+    private modal: ModalService,
+    private route: ActivatedRoute,
+    private router: Router
+  ) {}
+
+  ngOnInit() {
+    this.routeSub = this.route.params.subscribe(params => {
+     this.id = +params['id'];
```

Comment:
In this case, I think without that typescript was complaining about me assigning a string (angular parses route params as strings) to a number.